packages/ui/src/styles/globals.css

```diff
@@ -1,5 +1,6 @@
 @import "tailwindcss";
-
+@source "../../../../apps/";
+@source "..";
```

Comment:
What sorcery is this? :joy:

Reply:
https://github.com/shadcn-ui/ui/discussions/6486#discussioncomment-12128590

Fue lo que te dije de la integracion de shadcn con tailwind v4, por ahora fue la unica forma que pudo funcionar.

Reply:
Le puse un comment por ahora.